packages/opentelemetry-exporter-collector/src/platform/node/sendSpans.ts

```diff
@@ -70,7 +73,7 @@ export function sendSpans(
       },
       attributes: collectorExporter.attributes,
     },
-    // resource: '', not implemented
+    resource: resource.labels,
```

Comment:
I think this is wrong
the Resource type for collector is this:
https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-exporter-collector/src/types.ts#L270

Can you verify that you actually see the resource in collector ?

Reply:
I think the build would fail if the type mismatched

Reply:
> Can you verify that you actually see the resource in collector ?

I am going to run the example and verify the behavior.

Reply:
I tried the basic example with Collector + Zpkin Exporter, everything is working file. However, I don't see resource/attributes in zipkin UI because Collector Zipkin is not exporting resource data.

Reply:
but the interface shows that the resource is not label, why this
`resource: resource.labels`
instead of 
`resource: {labels: resource.labels, type: 'some type'}` ?

Reply:
@obecny thanks for catching this, I addressed in https://github.com/open-telemetry/opentelemetry-js/pull/843/commits/b7c7dc08e4ea97ca632bff757abe6a5522ba1c15. Somehow the compiler didn't catch that.